helm/blueapi/values.yaml

```diff
@@ -2,121 +2,189 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# May be required for communication with EPICS on the host node
+hostNetwork: false
+
+# Whether to restart the pod automatically on a change to any configMap
+# includes most changes to `tracing`, `worker` or `initContainer` below
+# Kubernetes does not treat a change to config only as need to restart a pod,
+# but we pick up configuration only on startup.
+restartOnConfigChange: true
+
+# Additional envVars to mount to the pod
+extraEnvVars: []
+# - name: RABBITMQ_PASSWORD
+#   valueFrom:
+#     secretKeyRef:
+#       name: rabbitmq-password  # this secret is defined in the RabbitMQ deployment
+#       key: rabbitmq-password
+
+# Configure tracing support
+tracing:
+  # Configure use of the Open Telemetry Protocol
+  otlp:
+    enabled: false
+    protocol: http/protobuf
+    server:
+      host: https://daq-services-jaeger # replace with central instance
+      port: 4318
+
+# Config for the worker goes here, will be mounted into a config file
+worker:
+  api:
+    host: 0.0.0.0 # Allow non-loopback traffic
+    port: 8000
+  env:
+    sources:
+      - kind: deviceFunctions
+        module: blueapi.startup.example_devices
+      - kind: planFunctions
+        module: blueapi.startup.example_plans
+      - kind: planFunctions
+        module: dodal.plans
+      - kind: planFunctions
+        module: dodal.plan_stubs.wrapped
+  stomp:
+    auth:
+      username: guest
+      password: guest
+    host: rabbitmq
+    port: 61613
+
+initContainer:
+  enabled: false
+  config:
```

Comment:
This is a breaking change, is it really necessary?

Reply:
In my opinion yes, config should be under its own key and not included with non-ApplicationConfig compatible keys, like enabled.

Reply:
No longer required due to other changes to how the scratch is loaded.